Added site to pss file and assigned 1 to site number so it is compatible...

apourmok opened the pull request:
Added site and site number 1 to pss file so it is compatible and ED can read it. This assumes that extraction is for only one site.

mdietze commented:
Does this code already write out a *.site file for each location? If not, then adding the site column seems like it would break more than it would fix. How to appropriately fill in all the data that's needed in the site file (topography, hydrological routing, soil texture, site clustering) opens a whole bag of worms that we need to deal with, but which I'd been putting off until we had met and veg under control.

apourmok commented:
I removed adding site per @mdietze request. Also I added new module for converting fia database to land cover as we discussed. Working on linking it to NLCD.

mdietze commented:
Doh, that broke everything -- I should have looked more closely at CanopyCover2NLCD, which is the offender. This needs to be a function, not a script, to live in the /R folder. For now please move it to the /inst folder and then, once you've had a chance to actually make this a generalized function, you can move it back later.
